src/main/resources/data/techreborn/recipes/centrifuge/coarse_dirt.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "techreborn:centrifuge",
+  "power": 5,
+  "time": 2500,
+  "ingredients": [
+    {
+      "item": "minecraft:coarse_dirt",
+      "count": 16
+    }
+  ],
+  "results": [
```

Comment:
This could be 8xSand + 2xGravel + Flint to be in line with dirt

Reply:
didn't think of that one. my idea was that coarse dirt has a lot more gravel in it and there is no centrifuge recipe for gravel. Maybe 8 Gravel, 2 Sand, 1 Flint?
Anyway, I do not have strong feelings about this recipe, do as you like

Reply:
Yeah, 8xGravel, 2xSand and Flint looks good